extensions/wikia/TrackingOptIn/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "dependencies": {
-    "@wikia/tracking-opt-in": "^1.0.32",
+    "@wikia/tracking-opt-in": "github:wikia/tracking-opt-in#c84c30bc31ad85c8e33cea378e9f9b818042dd39",
```

Comment:
Remember to update after https://github.com/Wikia/tracking-opt-in/pull/51 is released.